js/client-battle-tooltips.js

```diff
@@ -409,6 +427,349 @@ var BattleTooltips = (function () {
+								modifier = modifier * 2;
+							}
+						}
+					}
+					if (pokemon.volatiles) {
+						for (var volatile in pokemon.volatiles) {
+							if (volatile === 'slowstart') {
+								modifier = modifier * 0.5;
+							}
+							if (volatile === 'itemremoved' && pokemon.ability === 'Unburden') {
+								modifier = modifier * 2;
+							}
+						}
+					}
+					if (pokemon.item === 'Quick Powder') {
+						if (pokemon.baseSpecies === 'Ditto') {
+							if (!('transform' in pokemon.volatiles)) {
+								modifier = modifier * 2;
+							}
+						}
+					}
+
+				}
+
+			}
+		}
+
+		return modifier;
+	};
+
+	BattleTooltips.prototype.calculateModifiedStats = function (pokemon) {
+		var modifiedStats = {};
+		if (this.battle.gen === 1) {
+			modifiedStats = {atk: Math.max(Math.floor(pokemon.stats['atk'] * this.findModifier(pokemon, 'atk')), 999),
+							 def: Math.max(Math.floor(pokemon.stats['def'] * this.findModifier(pokemon, 'def')), 999),
+							 spa: Math.max(Math.floor(pokemon.stats['spa'] * this.findModifier(pokemon, 'spa')), 999),
+							 spe: Math.max(Math.floor(pokemon.stats['spe'] * this.findModifier(pokemon, 'spe')), 999)};
+
+		} else {
+			modifiedStats = {atk: Math.floor(pokemon.stats['atk'] * this.findModifier(pokemon, 'atk')),
+							 def: Math.floor(pokemon.stats['def'] * this.findModifier(pokemon, 'def')),
+							 spa: Math.floor(pokemon.stats['spa'] * this.findModifier(pokemon, 'spa')),
+							 spd: Math.floor(pokemon.stats['spd'] * this.findModifier(pokemon, 'spd')),
+							 spe: Math.floor(pokemon.stats['spe'] * this.findModifier(pokemon, 'spe'))};
+		}
+		return modifiedStats;
+	};
+
+	BattleTooltips.prototype.makeModifiedStatText = function (pokemon, modifiedStats) {
+		var statsText = '<p>';
+		var statTable = {atk: '&nbsp;Atk /&nbsp;', def: '&nbsp;Def /&nbsp;', spa: '&nbsp;SpA /&nbsp;',
+						 spc: '&nbsp;Spc /&nbsp;', spd: '&nbsp;SpD /&nbsp;', spe: '&nbsp;Spe</p>'};
+		statsText += this.boldModifiedStat(pokemon, modifiedStats, 'atk') + statTable['atk'];
+		statsText += this.boldModifiedStat(pokemon, modifiedStats, 'def') + statTable['def'];
+		statsText += this.boldModifiedStat(pokemon, modifiedStats, 'spa');
+		if (this.battle.gen === 1) {
+			statsText += statTable['spc'];
+		} else {
+			statsText += statTable['spa'];
+			statsText += this.boldModifiedStat(pokemon, modifiedStats, 'spd') + statTable['spd'];
+		}
+		statsText += this.boldModifiedStat(pokemon, modifiedStats, 'spe') + statTable['spe'];
+		return statsText;
+	};
+
+	BattleTooltips.prototype.boldModifiedStat = function (pokemon, modifiedStats, statName) {
+		var statText = '';
+		if (pokemon.stats[statName] === modifiedStats[statName]) {
+			statText += '' + modifiedStats[statName];
+		} else {
+			statText += '<b>' + modifiedStats[statName] + '</b>';
+		}
+		return statText;
+	};
+
+	BattleTooltips.prototype.checkEviolite = function (pokemon) {
+		if (pokemon.tier === 'LC' || pokemon.tier === 'LC Uber' || pokemon.tier === 'NFE') {
```

Comment:
It actually is! I noticed that it was a field when I was looking around the heap snapshots I made in the JavaScript console. As a test when I was writing this feature, I told Tooltip to add a line to each Pokemon's Tooltip which included it's tier. Sure enough, it worked. Checking to see if the tier was 'LC', 'LC Ubers', and 'NFE' improved performance over checking the giant table of NFE mons I made. However, your suggestion to use template.evos should make this improvement obsolete.